modules/Bio/EnsEMBL/Production/Pipeline/FASTA/CopyNCBIBlastDNA.pm

```diff
@@ -106,12 +106,27 @@ sub get_blast_files {
   my $base = $self->param('ftp_dir');
   my $type = $self->param('type');
   my $old_path = File::Spec->catdir($base, "release-$old_release", 'ncbi_blast', $type);
+  my $repeatmask_value = $self->get_repeatmask_value($self->param('species'));
   my $filter = sub {
     my ($filename) = @_;
-    return ($filename =~ /$species\./) ? 1 : 0;
+    return ($filename =~ /$species\./ && $filename =~ /$repeatmask_value\.dna/) ? 1 : 0;
```

Comment:
BTW shouldn't it be a `or` statement instead?

Reply:
file name matches both species names and repeat masker date : 
ex: Homo_sapiens.GRCh38.20141216.dna
species name : Homo_sapiens and repeatmasker date: 20141216

Reply:
ok fine! Thanks